### ISSUE TITLE
UnusedPrivateElements: false positive with constant in string

### ISSUE THREAD

**Issue by janedbal** [slevomat/coding-standard#367]
Following code:
```php
class Foo {
    private const FOO = 1;

    public function test() {
        return "{$this->foo(self::FOO)}";
    }
}
```

results in `Class Foo contains unused private constant FOO` error.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/98fa7a32590be759da96cd63122e4bc1f7828f07

**Comment by janedbal** [comment]
Thanks!
